### PR TITLE
ipfix: add ipfix_report_interval field to NWI

### PR DESCRIPTION
**Pull request by ivan4th**
The field specifies minimum per-flow IPFIX reporting interval in
seconds. If the field is set to 0 or ~0, the default value of 5
seconds is used. It's possible to set the value both through the
binapi and CLI.
Also, resolve test issues from #255

### CODE REVIEW

File: upf/upf.api [hunk 1]
```diff
@@ -237,6 +237,7 @@ autoreply define upf_nwi_add_del {
   u32 ip6_table_id;
   u8 ipfix_policy[64];
   vl_api_address_t ipfix_collector_ip;
+  u32 ipfix_report_interval;
```
Comment (sergeymatov) [inline upf/upf.api:240]: So, this IPFIX-related fields won't be added into nwi_details?

File: upf/upf_ipfix.c [hunk 2]
```diff
@@ -46,11 +46,14 @@
   do { } while (0)
 #endif
 
+/* Default report interval in seconds */
+#define UPF_IPFIX_DEFAULT_REPORT_INTERVAL 5
```
Comment (sergeymatov) [inline upf/upf_ipfix.c:50]: why it's traveled from header file to here?

**Review by sergeymatov — APPROVED** [review]
LGTM overal

File: upf/upf.api (hunk 1 above)
Comment (ivan4th) [reply upf/upf.api:240]: Thanks, forgot to add them there. Will post an update shortly

File: upf/upf_ipfix.c (hunk 2 above)
Comment (ivan4th) [reply upf/upf_ipfix.c:50]: It's not used outside this C file, and there's no reason to have almost whole UPG recompiled when one changes this constant

File: upf/upf.api (hunk 1 above)
Comment (ivan4th) [reply upf/upf.api:240]: Added

**Pull request merged**